04-advanced-topics/network-analysis_rnaseq_01_wgcna.Rmd

```diff
@@ -26,156 +26,79 @@ As with many clustering and network methods, there are some parameters that may
+To obtain this data, go to the [dataset's page on refine.bio](https://www.refine.bio/experiments/SRP133573/identification-of-transcription-factor-relationships-associated-with-androgen-deprivation-therapy-response-and-metastatic-progression-in-prostate-cancer) and click the "Download Now" button. 
+Follow the instructions, being sure to select "Skip quantile normalization for RNA-seq samples".
 
-## Set up your analysis folders 
+Once you have downloaded and unzipped the refine.bio dataset, place the `SRP133573` folder in a `data` subdirectory of your analysis folder.
 
-Good file organization is helpful for keeping your data analysis project on track!
-We have set up some code that will automatically set up a folder structure for you. 
-Run this next chunk to set up your folders! 
+You will also want to create `plots` and `results` folders for future use. 
+At this point, your analysis directory should contain:
 
-If you have trouble running this chunk, see our [introduction to using `.Rmd`s](https://alexslemonade.github.io/refinebio-examples/01-getting-started/getting-started.html#how-to-get-and-use-rmds) for more resources and explanations. 
-
-```{r}
-# Create the data folder if it doesn't exist
-if (!dir.exists("data")) {
-  dir.create("data")
-}
-
-# Define the file path to the plots directory
-plots_dir <- "plots" # Can replace with path to desired output plots directory
-
-# Create the plots folder if it doesn't exist
-if (!dir.exists(plots_dir)) {
-  dir.create(plots_dir)
-}
-
-# Define the file path to the results directory
-results_dir <- "results" # Can replace with path to desired output results directory
-
-# Create the results folder if it doesn't exist
-if (!dir.exists(results_dir)) {
-  dir.create(results_dir)
-}
-```
-
-In the same place you put this `.Rmd` file, you should now have three new empty folders called `data`, `plots`, and `results`!
-
-## Obtain the dataset from refine.bio
-
-For general information about downloading data for these examples, see our ['Getting Started' section](https://alexslemonade.github.io/refinebio-examples/01-getting-started/getting-started.html#how-to-get-the-data). 
-
-Go to this [dataset's page on refine.bio](https://www.refine.bio/experiments/SRP133573/identification-of-transcription-factor-relationships-associated-with-androgen-deprivation-therapy-response-and-metastatic-progression-in-prostate-cancer).
-
-Click the "Download Now" button on the right side of this screen.  
-
-<img src="https://github.com/AlexsLemonade/refinebio-examples/raw/40e47f4d3f39283effbd9843a457168061be9680/template/screenshots/download-now.png" width=200>  
-
-Fill out the pop up window with your email and our Terms and Conditions:  
-
-<img src="https://github.com/AlexsLemonade/refinebio-examples/raw/40e47f4d3f39283effbd9843a457168061be9680/template/screenshots/download-email.png" width=500>  
-
-We are going to use non-quantile normalized data for this analysis.
```

Comment:
I'm worried about deleting this piece in particular. We need them to know whether or not they should download quantile normalized data and where to find that.

Reply:
I do cover that on line 40, but it could be made more prominent.

Reply:
I think screenshots help. I guess bigger question is just because someone's an "advanced topics" user, can we assume they know how to download data from refine.bio and know the refine.bio options more readily?

Reply:
I guess I think even advanced topics users will appreciate screenshots. Though I do agree with cutting back on the file path hand holding.

Reply:
That was my thought, or they could go to another example to get this information. 🤷🏼

Reply:
> go to another example to get this information. 🤷🏼

If we think they might do this, then we should probably just keep the screenshots here too. I think screenshots help decrease brain glucose usage for simple things.